README.md

```diff
@@ -1181,4 +1181,4 @@ See [CONTRIBUTING](CONTRIBUTING.md).
 
 ## Copyright and License
 
-Copyright (c) 2012-2014 Tim Vandecasteele and contributors. See [LICENSE.txt](LICENSE.txt) for details.
+Copyright (c) 2012-2016 Tim Vandecasteele and contributors. See [LICENSE.txt](LICENSE.txt) for details.
```

Comment:
IANAL, but I think we can't just strip copyright from @tim-vandecasteele like that ;)